sprockets_postgres.py

```diff
@@ -355,8 +350,12 @@ async def postgres_connector(self,
                             _attempt + 1) as connector:
                         yield connector
                     return
-                message = 'disconnected'
-            exc = on_error('postgres_connector', ConnectionException(message))
+            # Handle timeout error during cursor creation as
+            # operational error
+            elif isinstance(err, asyncio.TimeoutError):
+                exc = on_error(
```

Comment:
As we discussed, I'm not too happy about needing this, but ok compromise IMO. But looking now, I think you meant 
```
elif isinstance(err, asyncio.TimeoutError):
    exc = psycopg2.OperationalError(str(err))
exc = on_error('postgres_connector', err)
```

Reply:
ok got it, will also wait for @gmr and wrap err as string

Reply:
I don't think that's correct, in the example shown `exc` is reassigned on the next line.